pulp_python/app/viewsets.py

```diff
@@ -156,13 +156,15 @@ def sync(self, request, pk):
         )
         serializer.is_valid(raise_exception=True)
         repository = serializer.validated_data.get('repository')
+        mirror = serializer.validated_data.get('mirror', True)
```

Comment:
This value defaults to `True` in core, and that serializer is how this option is documented. So I think we shouldn't default here, since our default wouldn't be connected to the schema.
https://github.com/pulp/pulp/blob/cdf6133394d99bbc21a69fae4599d8271d6b344c/pulpcore/pulpcore/app/serializers/repository.py#L142